CHANGELOG-LATEST.md

```diff
@@ -1,8 +1,10 @@
-## 3.1.0
+## 3.1.1
 
-- iOS: 
+- Fixed an issue in the example app with null-safety enabled.
```

Comment:
I would clarify here saying that the error that it fixes is: Unhandled Exception: type 'Null' is not a subtype of type 'String' in type cast

The exception is not specific to our sample app

Reply:
good call, updated